Wristimate/src/Behaviour.cs

```diff
@@ -130,9 +132,9 @@ private IEnumerable<MagData> Magazines()
 				{
 					var trans = h.transform;
 					var interactable = h.CurrentInteractable;
-					var mag = (interactable != null ? interactable.GetComponent<FVRFireArmMagazine>() : null)!;
+					var mag = (interactable?.GetComponent<FVRFireArmMagazine>())!;
```

Comment:
This is not the same as the line above. Null conditionals don't work on Unity objects.